src/main/java/racingcar01/domain/Distance.java

```diff
@@ -0,0 +1,32 @@
+package racingcar01.domain;
+
+public class Distance {
+    private static final String MIN_DISTANCE_MASSAGE = "최소 거리는 0 입니다.";
+    private static final int MIN_DISTANCE = 0;
+    private static final int ADD_DISTANCE = 1;
+
+    private final int distance;
+
+    private Distance(int distance) {
+        this.distance = distance;
+    }
+
+    public static Distance of(int distance) {
+        validMinDistace(distance);
+        return new Distance(distance);
+    }
+
+    public Distance add() {
+        return of(distance + ADD_DISTANCE);
+    }
+
+    public int distance() {
+        return distance;
+    }
```

Comment:
```suggestion
    public int toInt() {
        return distance;
    }
```
또는 value() 이름으로 원시값 반환해보자.